Source/CBLRestPusher.m

```diff
@@ -560,7 +560,10 @@ - (BOOL) uploadMultipartRevision: (CBL_Revision*)rev {
                           _dontSendMultipart = YES;
                           [self uploadJSONRevision: rev];
                       } else {
-                          self.error = error;
+                          // 403/Forbidden means validation failed; don't treat it as an error
+                          // because I did my job in sending the revision.
+                          if (error.code != kCBLStatusForbidden)
```

Comment:
Should this follow the same logic as the equivalent code in -uploadBulkDocs:? That code also treats `kCBLStatusUnauthorized` the same way. And for both of them it does not call `[self revisionFailed]`, because that would result in retrying the document next time.